docs/topics/creating_new_clusters.md

```diff
@@ -0,0 +1,166 @@
+# Creating Kubernetes Clusters
+
+## Deploy
+
+The `aks-engine deploy` command will create a new Kubernetes cluster from scratch into a pre-existing Azure resource group. You define an API model (cluster definition) as a JSON file, and then pass in a reference to it, as well as appropriate Azure credentials, to a command statement like this:
+
+```sh
+$ aks-engine deploy --subscription-id $SUBSCRIPTION_ID \
+    --dns-prefix $CLUSTER_NAME \
+    --resource-group $RESOURCE_GROUP \
+    --location $LOCATION \
+    --api-model examples/kubernetes.json \
+    --client-id $SERVICE_PRINCIPAL_ID \
+    --client-secret $SERVICE_PRINCIPAL_PASSWORD
+```
+
+`aks-engine deploy` is a long-running operation that creates Azure resources (e.g., Virtual Machine and/or Virtual Machine Scale Set (VMSS), Disk, Network Interface, Network Security Group, Public IP Address, Virtual Network, Load Balancer, and others) that will underly a Kubernetes cluster. All deployed VMs will be configured to run Kubernetes bootstrap scripts appropriate for the desired cluster configuration. The outcome of a successful `aks-engine deploy` operation is a fully operational Kubernetes cluster, ready for use immediately.
```

Comment:
The general recommendation for nested parentheses in English is to use square brackets for the inner pair, such as:

```
(e.g., Virtual Machine Scale Set [VMSS], Disk, and others) that...
```

But whatevs, I think it's ok as-is.